src/config.rs

```diff
@@ -195,6 +195,9 @@ pub struct GooseConfiguration {
     /// Follows base_url redirect with subsequent requests
     #[options(no_short)]
     pub sticky_follow: bool,
+    /// Allows to test internally when https validation is not required
```

Comment:
How about documenting this as:
`Disables validation of https certificates`

We can better document in the goose book (and it can happen in a followup PR).